2019-11-27 20:25:41,762 ERROR c.d.c.f.Reader [main] An error occurred while listing existing files.
java.nio.file.NoSuchFileException: /home/ilegra/data/in
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86) ~[?:1.8.0_222]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102) ~[?:1.8.0_222]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107) ~[?:1.8.0_222]
	at sun.nio.fs.UnixFileSystemProvider.newDirectoryStream(UnixFileSystemProvider.java:427) ~[?:1.8.0_222]
	at java.nio.file.Files.newDirectoryStream(Files.java:457) ~[?:1.8.0_222]
	at java.nio.file.Files.list(Files.java:3451) ~[?:1.8.0_222]
	at com.deividsantos.challenge.file.Reader.listAlreadyExistentFiles(Reader.java:33) [main/:?]
	at com.deividsantos.challenge.service.EventService.processAlreadyExistentFiles(EventService.java:20) [main/:?]
	at com.deividsantos.challenge.SalesMain.main(SalesMain.java:17) [main/:?]
2019-11-27 20:25:41,786 ERROR c.d.c.f.Watcher [pool-2-thread-1] An error occurred while setting the file directory.
java.nio.file.NoSuchFileException: /home/ilegra/data/in
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86) ~[?:1.8.0_222]
	at sun.nio.fs.UnixException.asIOException(UnixException.java:111) ~[?:1.8.0_222]
	at sun.nio.fs.LinuxWatchService$Poller.implRegister(LinuxWatchService.java:246) ~[?:1.8.0_222]
	at sun.nio.fs.AbstractPoller.processRequests(AbstractPoller.java:260) ~[?:1.8.0_222]
	at sun.nio.fs.LinuxWatchService$Poller.run(LinuxWatchService.java:364) ~[?:1.8.0_222]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
